src/state/connection-mixin.ts

```diff
@@ -73,13 +73,14 @@ export const connectionMixin = <T extends Constructor<HassBaseEl>>(
         translationMetadata,
         dockedSidebar: "docked",
         vibrate: true,
+        debugConnection: false,
```

Comment:
I think we should store this in `localStorage`, now you will never be able to catch things when the page is refreshed?

Reply:
Does including this in ha-pref-storage take care of that? I thought it was working when I tested it.

Reply:
It looks like I'm getting everything I expect to see when hitting refresh with this toggle enabled (and I did remove the __DEV__ part of the equation just to test and be sure).

![image](https://github.com/home-assistant/frontend/assets/32912880/61416dd9-ec29-42f8-983a-9a2f430244bb)

Reply:
Oh right, missed that 🙈 yeah pref storage stores that part in local storage!